patterns/grok-patterns

```diff
@@ -88,7 +88,9 @@ QS %{QUOTEDSTRING}
 # Log formats
 SYSLOGBASE %{SYSLOGTIMESTAMP:timestamp} (?:%{SYSLOGFACILITY} )?%{SYSLOGHOST:logsource} %{SYSLOGPROG}:
 COMMONAPACHELOG %{IPORHOST:clientip} %{USER:ident} %{USER:auth} \[%{HTTPDATE:timestamp}\] "(?:%{WORD:verb} %{NOTSPACE:request}(?: HTTP/%{NUMBER:httpversion})?|%{DATA:rawrequest})" %{NUMBER:response} (?:%{NUMBER:bytes}|-)
-COMBINEDAPACHELOG %{COMMONAPACHELOG} %{QS:referrer} %{QS:agent}
+COMBINEDAPACHELOG %{COMMONAPACHELOG} "%{QS:referrer}" "%{QS:agent}"
```

Comment:
You don't need the quotes around this.  QS refers to "Quoted String," and it is meant to keep the quotes around it.

Changing this would break configurations for a great many people.  The correct way to strip quotes from around a quoted string would be to use mutate gsub.